### PR TITLE
Various improvements in our release instructions and scripts

### PR DESCRIPTION
**Pull request by apyrgio**
Make the following improvements to our release instructions and scripts:
1. Add a way to check if the installed Python version is the expected one.
2. Install the Thunar file manager in our Dangerzone container environments.
3. Do not use the builder cache by default, when building the Dangerzone container image. Add instead the `--use-cache` flag, when the developer wants to expedite the process for non-release builds.

### CODE REVIEW

**Review by legoktm — APPROVED** [review]
LGTM

**Comment by apyrgio** [comment]
For this particular change, I'll wait for our CI to become green again before I merge. Thanks for the review Kunal!

**Pull request merged**
